backend/generator.js

```diff
@@ -4,11 +4,11 @@ var mailer = require('./mailer');
 var uuidV4 = require("uuid/v4");
 var validation = require("../public/scripts/validation.js");
 var spawn = require('child_process').spawn;
-var output = require('../util/output');
+var socketHandler = require('./socketHandler.js');
 
-exports.executeScript = function (socket, formData) {
+exports.executeScript = function (socket, formData, cb) {
```

Comment:
just a js standard where people widely use. https://stackoverflow.com/questions/31780872/what-is-cb-in-node Anyways I made changes